build/lib/common.sh

```diff
@@ -76,3 +76,30 @@ function build::gather_licenses() {
      -o -name 'NOTICE' -o -name 'COPYING' -o -name 'NOTICE.txt' \
      -o -name 'LICENSE.code' -o -name 'LICENCE.md' \) | while IFS= read -r NAME; do mkdir -p "${outputdir}/$(dirname $NAME)" && cp  "$NAME" "${outputdir}/${NAME}"; done
 }
+
+function build::common::use_go_version() {
```

Comment:
Minor nit, it would be nice if when adding golang versions to builder base, we created a symlink to like /what/ever/bin/go1.14.13 -> go1.14, that way in this repo we don't have to know the specific patch versions for each minor version.

Reply:
I get what you are saying, that's a good idea. I would be creating the symlinks on the install side so would need to create another PR for the build-tooling repo but I can do that.